src/utilities/mbclean.c

```diff
@@ -81,1118 +81,1073 @@
 
 /* ping structure definition */
 struct mbclean_ping_struct {
-  int time_i[7];
-  double time_d;
-  int multiplicity;
-  double navlon;
-  double navlat;
-  double speed;
-  double heading;
-  int beams_bath;
-  char *beamflag;
-  char *beamflagorg;
-  double *bath;
-  double *bathacrosstrack;
-  double *bathalongtrack;
-  double *bathx;
-  double *bathy;
+	int time_i[7];
+	double time_d;
+	int multiplicity;
+	double navlon;
+	double navlat;
+	double speed;
+	double heading;
+	int beams_bath;
+	char *beamflag;
+	char *beamflagorg;
+	double *bath;
+	double *bathacrosstrack;
+	double *bathalongtrack;
+	double *bathx;
+	double *bathy;
 };
 
 /* bad beam identifier structure definition */
 struct bad_struct {
-  int flag;
-  int ping;
-  int beam;
-  double bath;
+	int flag;
+	int ping;
+	int beam;
+	double bath;
 };
 
-static const char program_name[] = "mbclean";
```

Comment:
@dwcaress Why did you remove these changes?  Large string constants work better as static const and being in the global scope.

Reply:
I'll put that back. You and I both made changes to mbclean.c, and I had difficulty merging the changes.

Reply:
ok.  no worries.  If you don't already know it, check out [meld](https://meldmerge.org).  It makes merging files easier some of the time